docs/installation/Migrating-to-Python3

```diff
@@ -0,0 +1,27 @@
+# Migration
+Remove Python 2 packages
+```no-highlight
+# apt-get remove --purge -y python-dev python-pip libxml2-dev libxslt1-dev libffi-dev graphviz libpq-dev libssl-dev zlib1g-dev
```

Comment:
Uninstalling development packages here and reinstalling them below doesn't quite make sense.